app/views/ReportView/components/CopyNumber/columnDefs.ts

```diff
@@ -34,17 +37,28 @@ const columnDefs = [{
   hide: false,
 }, {
   headerName: 'Expression (RPKM)',
-  field: 'gene.expressionVariants.rpkm',
-  hide: false,
+  colId: 'rpkm',
+  valueGetter: 'data.gene.expressionVariants.rpkm',
+  hide: true,
 }, {
   headerName: 'Expression (Normal FC)',
   colId: 'primarySiteFoldChange',
-  field: 'gene.expressionVariants.primarySiteFoldChange',
+  valueGetter: 'data.gene.expressionVariants.primarySiteFoldChange',
```

Comment:
Similar to TPM vs RPKM we want to show Expression (Normal FC) OR Expression (kIQR) depending on what's available

Reply:
just for copy number or for all 3 of these?